pi5/board.go

```diff
@@ -439,7 +444,108 @@ func (b *pinctrlpi5) StreamTicks(ctx context.Context, interrupts []board.Digital
 	return nil
 }
 
+func (b *pinctrlpi5) configureBT(cfg *rpiutils.Config) error {
+	var configChanged bool = false
+	var configFailed bool = false
+	var err error
+	configPath := rpiutils.GetBootConfigPath()
+
+	// Handle enable_uart
+	if cfg.BoardSettings.BTenableuart != nil {
+		b.logger.Debugf("cfg.BoardSettings.BTenableuart=%v", *cfg.BoardSettings.BTenableuart)
+
+		if *cfg.BoardSettings.BTenableuart == true {
+			// remove any previous enable_uart=0 settings
+			configChanged, err = rpiutils.RemoveConfigParam(configPath, "enable_uart=0", b.logger)
+			if err != nil {
+				b.logger.Errorf("Failed to modify Bluetooth settings in boot config: %v", err)
```

Comment:
different error messages please -- identical error strings make it very hard to debug.

Reply:
If there's an error manipulating the config.txt file, I don't really think that some bluetooth parameter changes will succeed and others will fail.  In all cases, the log will include Info/Debug messages that report up until the Error message.  It would be clear from reading the log where it goes sideways.  Really no need to come up with unique error message strings.

Reply:
I disagree. I won't block on this comment, but as someone who will likely debug these if it happens in the future, knowing clearly what updates succeeded and failed by looking at the logs alone is extremely helpful. Note that debug logs are typically filtered out.